src/style.css

```diff
@@ -0,0 +1,102 @@
+/* stylelint-disable indentation */
+/* stylelint-disable CssSyntaxError */
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+}
+
+.image {
+  display: block;
+  position: absolute;
+  margin-left: 26px;
+  margin-top: 25px;
+  width: 19px;
+  height: 19px;
+}
+
+.img-s {
+  height: 11px;
+  width: 11px;
+  margin-left: 13px;
+  margin-top: 15px;
+}
+
+/* .form {
+  font-family: inherit;
+  font-weight: 800;
+  width: 100%;
+  display: block;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  padding-left: 62px;
+  height: 70px;
+  font-size: 16px;
+  line-height: 22px;
+} */
+
+/* .form-b {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.form-s {
+  padding-left: 30px;
+  height: 42px;
+  font-size: 13px;
+  line-height: 19px;
+} */
+.input {
+  display: block;
+  font-weight: 300;
+  width: 100%;
+  height: 70px;
+  font-size: 16px;
+  font-family: "Avenir-Book", sans-serif;
+  margin-top: 20px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  padding-left: 62px;
+}
+
+.input-s {
+  height: 42px;
+  font-size: 14px;
+  font-family: "Avenir-Book", sans-serif;
+  margin-top: 20px;
+  padding-left: 33px;
+}
+
+.input:hover {
+  background: #ffff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.form-b:focus {
```

Comment:
what is form-b?

Also no need to create additional classes, you can move this styles under general `form `class and modify it for small form in `form--small `class